src/Component/Menu/User/User.tsx

```diff
@@ -55,8 +55,20 @@ export const User: React.FC<UserProps> = (props) => {
         client.getKeycloak()?.accountManagement();
         break;
       case 'logout':
-        // TODO: Fix logout for non keycloak setups
-        client.getKeycloak()?.logout();
+        const keycloak = client.getKeycloak();
+        if (keycloak) {
+          keycloak.logout();
+        } else {
+          fetch('/auth/logout', {
+            method: 'POST',
+            credentials: 'same-origin'
+          }).then(response => {
+            if (response.status === 200) {
+              localStorage.removeItem('gpro-auth-token');
```

Comment:
This one is out of place here.. Can we provide some kind of callback after successful logout or similar instead?

Or at least to make the token name configurable via config to get less project specific

Reply:
Good point, I amended this to make it configurable.